components/contribution-flow/ContributionFlowStepContainer.js

```diff
@@ -100,8 +98,6 @@ class ContributionFlowStepContainer extends React.Component {
             profiles={this.props.contributeProfiles}
             collective={collective}
             stepDetails={stepDetails}
-            defaultEmail={this.props.defaultEmail}
-            defaultName={this.props.defaultName}
```

Comment:
No need to pass them down anymore; the default state is set in the parent component (index) through `getDefaultStepProfile`